HotelManagement/src/main/java/com/deltasource/hotelmanagement/core/Hotel.java

```diff
@@ -27,21 +42,33 @@ public String getName() {
 
 	/**
 	 * Sets the name of the hotel.
+	 *
 	 * @param name the name to be set
 	 */
 	private void setName(String name) {
 		if (name == null) {
```

Comment:
you can do both checks in one if statement and use some ingenuity for the exception message :)